python/nano/src/bigdl/nano/deps/ipex/ipex_api.py

```diff
@@ -50,6 +50,22 @@ def PytorchIPEXJITModel(model, input_sample=None, use_ipex=False,
                                use_jit=use_jit, channels_last=channels_last)
 
 
+def PytorchIPEXJITBF16Model(model, input_sample=None, use_ipex=False,
```

Comment:
Maybe it is better to use lower case separated with an underscore for function names.

Reply:
Maybe we should open another PR to resolve the function name issue? I think many of them should be modified in `deps`.